testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/interceptor/annotatedejbclient/ClientInterceptor.java

```diff
@@ -0,0 +1,54 @@
+package org.jboss.as.test.integration.ejb.interceptor.annotatedejbclient;
+
+import java.util.logging.Logger;
+
+import org.jboss.ejb.client.EJBClientInterceptor;
+import org.jboss.ejb.client.EJBClientInvocationContext;
+
+/**
+ * Client side JBoss interceptor.
+ */
+public class ClientInterceptor implements EJBClientInterceptor {
+
+  private Logger log = Logger.getLogger(ClientInterceptor.class.getName());
+
+  /**
+   * Creates a new ClientInterceptor object.
+   */
+  public ClientInterceptor() {
+  }
+
+  @Override
+  public void handleInvocation(EJBClientInvocationContext context) throws Exception {
+    
+    log.info("In the client interceptor handleInvocation : " + this.getClass().getName() + " " + context.getViewClass() + " " + context.getLocator());
```

Comment:
Logging should be done at debug level or lower, normal test execution should produce no log output.